app/views/alchemy/admin/pages/fold.js.erb

```diff
@@ -1,3 +1 @@
-$('#page_<%= @page.id -%>').replaceWith('<%= j render("page", page: @page) %>');
-Alchemy.watchForDialogs('#page_<%= @page.id %>');
-Alchemy.Spinner.watch('#page_<%= @page.id %>');
+Alchemy.Sitemap.fetch();
```

Comment:
We should not fetch the whole tree while folding/unfolding pages. 

We should only fetch the sub tree of unfolded page and hide the sub tree if a page gets folded.